identity/identity.go

```diff
@@ -507,6 +507,80 @@ func (i *Identity) WithDeclassifiedCredentials(ctx context.Context, c cipher.Pro
 	return &ii, nil
 }
 
+func (i *Identity) deleteCredentialWebAuthFromIdentity() error {
```

Comment:
This function was moved here from `identity/handler.go` where it had the signature `func deletCredentialWebAuthFromIdentity(identity *Identity) (*Identity, error)`